release-notes/10.0/preview/preview1/winforms.md

```diff
@@ -1,14 +1,75 @@
 
-- [Feature](#feature)
+For in-depth discussion of the APIs involved, and usage examples, please see the original [API Proposal](https://github.com/dotnet/winforms/issues/12362) in the WinForms repo.
 
-Windows Forms updates in .NET 10:
+## Obsoleted APIs
 
-- [What's new in Windows Forms in .NET 10](https://learn.microsoft.com/dotnet/desktop/winforms/whats-new/net100) documentation
+In .NET 10, WinForms has marked as obsolete the following APIs to help warn developers that BinaryFormatter may be used to deserialize data.
 
+**Clipboard:**
 
-## Feature
+- `public static object? GetData(string format)`
 
-This is something about the feature
+**DataObject:**
+
+- `public virtual object? GetData(string format)`
+- `public virtual object? GetData(string format, bool autoConvert)`
+- `public virtual object? GetData(Type format)`
+
+**ClipboardProxy (VisualBasic wrapper for WinForms Clipboard):**
+
+- `public object GetData(string format)`
+
+## New APIs
+
+We’ve introduced the following APIs in Windows Forms for .NET 10 Preview 1:
+
+**Clipboard:**
+
+- `public static void SetDataAsJson<T>(string format, T data)`
+- `public static bool TryGetData<T>(string format, out T data)`
+- `public static bool TryGetData<T>(string format, Func<Reflection.Metadata.TypeName, Type> resolver, out T data)`
+
+**DataObject:**
+
+- `public void SetDataAsJson<T>(T data)`
+- `public void SetDataAsJson<T>(string format, T data)`
+- `public void SetDataAsJson<T>(string format, bool autoConvert, T data)`
+- `public bool TryGetData<T>(out T data);`
+- `public bool TryGetData<T>(string format, out T data);`
+- `public bool TryGetData<T>(string format, bool autoConvert, out T data);`
+- `public bool TryGetData<T>(string format, Func<Reflection.Metadata.TypeName, Type> resolver, bool autoConvert, out T data);`

```

Comment:
Do we have to give the full namespace name? `Reflection.Metadata.TypeName` it's less readable this way, if people want details they will dig further.

Reply:
Updated